segtypes/common/codesubsegment.py

```diff
@@ -140,12 +152,15 @@ def process_insns(
                     sym_address, tryPlusOffset=False
                 )
                 if context_sym is not None:
-                    sym = self.parent.create_symbol(
-                        sym_address, offsets=True, reference=True
+                    # TODO: some way to determine if the symbol is in segment
+                    in_segment = context_sym.overlayCategory == self.get_most_parent().get_exclusive_ram_id()
```

Comment:
this doesn't make sense to me. many segments share the same ram id, so this will return lots of false positives. there's other parts of the codebase that check if something is in the segment, and I recommend using that approach.